### DISCUSSION TITLE
Usage/importance of server folder in DiME

### DISCUSSION THREAD

**Discussion by JunejaAmmy** [CURENT/andes#303] [General]
Hi Everyone,

I'm working on DiME to stream data for ANDES. Does someone know, what is the importance / usage / purpose of below `server` folder?
https://github.com/TheHashTableSlasher/dime/tree/master/server

I did post question in DiME github repository, but haven't received any response yet.

**Comment by cuihantao** [comment]
The `server` folder contains the source files for the server. DiME uses a server/client architecture, so the server needs to be up and running before the clients can connect.

**Comment by JunejaAmmy** [comment]
Many thanks Dr. Cui.

I have another query regarding server folder. I took below content from DiME Readme. I need further assistance to compile/use the server folder files. 

```
### Server

To compile the server executable, run `make` in the `server` directory. Build options can be tweaked by editing the Makefile (sane defaults are provided). The server code has the following compile-time dependencies:

* [Jansson](https://digip.org/jansson/)
* [OpenSSL](https://www.openssl.org/)
* [zlib](http://zlib.net/)
```

Your help will be really appreciable.

**Reply by cuihantao** [reply]
You should search on Google and learn by yourself how to compile a library in C. I'm not able to provide further assistance on this topic.
